superbuild/projects_modules/music-plugins.cmake

```diff
@@ -58,8 +58,6 @@ function(music_plugins_project)
             -DTETGEN_DIR:FILEPATH=${tetgen_DIR}
             -DQUAZIP_DIR:FILEPATH=${quazip_DIR}
             -DQUAZIP_INCLUDE_DIR:FILEPATH=${quazip_INCLUDE_DIR}
-            -DOPENSSL_ROOT_DIR:FILEPATH=${OPENSSL_ROOT_DIR}
-            -DZLIB_DIR:FILEPATH=${zlib_DIR}
```

Comment:
Is zlib a prerequisite ? I thought it was an external project.
If it's an external project, this line must not be removed, as music-plugins needs the path to find the lib.

Reply:
Indeed it's an external project, i had a warning saying that this parameter was never used. Maybe it's because Zlib is only used in medInria core, and not in music-plugins? There is however a `find_package(ZLIB REQUIRED)` in mscSRCClient, but i did not have any compilation or packaging problem with it. Weird

Reply:
It was never used in music-plugins, which is a bug. Some plugins were using find_package() as you said, but we should make sure we use 'our' zlib and not the system's, as we build quazip (another ep) with our zlib.
I'm trying to fixing this now.

Reply:
Oki, i am going to put back the use of ZLIB_DIR here for now then